src/docs.ts

```diff
@@ -27,8 +27,9 @@ export class Docs {
           return {
             design,
             doc: {
+              _id: uuid(),
```

Comment:
Seemed like we could automatically do this here for any docs that did not have a provided `_id` value.